docs/index.md

```diff
@@ -3,17 +3,17 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# ContactCS
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```

Comment:
Perhaps the websites here can be changed to ``https://github.com/AY2425S1-CS2103T-F12-1/tp/actions`` and ``https://app.codecov.io/gh/AY2425S1-CS2103T-F12-1/tp``? 
So it will be directed to ours instead of the original.

Reply:
I was aware of this part, but honestly this part should be Tianyi's job(as mentioned by Wei Seng in the group). Indeed, a lot of the stuff in this week's project is messy and overlapped, but I believe Tianyi can handle this part well so I would not make a change over here myself